### DISCUSSION TITLE
Motivation for dark selectors?

### DISCUSSION THREAD

**Discussion by boar-is** [irsyadadl/intentui#242] [Q&A]
`buttonStyles` have the following `shape` property:
```ts
shape: {
  square:
    'rounded-lg before:rounded-[calc(theme(borderRadius.lg)-1px)] after:rounded-[calc(theme(borderRadius.lg)-1px)] dark:after:rounded-lg',
  circle:
    'rounded-[9999px] before:rounded-[9998px] after:rounded-[9998px] dark:after:rounded-[9999px]',
},
```
What's the point of `after:rounded-[9998px] dark:after:rounded-[9999px]` and `after:rounded-[calc(theme(borderRadius.lg)-1px)] dark:after:rounded-lg`. Looks like an escape hatch for something. But I can't understand for what.

**Comment by irsyadadl** [comment]
In version 2 there's nothing anymore.
